src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -252,9 +252,10 @@ object ScalafixPlugin extends AutoPlugin {
                   val scalaV = scalaVersion.value
                   val msg = s"The SemanticDB scalac plugin version ${rev} set up " +
                     "via `semanticdbVersion` does follow the version recommended " +
-                    "for Scalafix, but is not supported for the outdated Scala " +
-                    s"version ${scalaV}. Please upgrade to a more recent Scala " +
-                    "patch version or uninstall sbt-scalafix."
+                    "for Scalafix, but is not supported for the given Scala " +
+                    s"version ${scalaV}. Please consider upgrading to a more recent version " +
+                    "of sbt-scalafix and/or Scala, or uninstalling sbt-scalafix plugin. " +
+                    "Crucially, sbt-scalafix must be published for the given Scala version."
```

Comment:
Can you clarify why you think that particular line can be beneficial? I would suggest to simply remove it

```suggestion
                    "of sbt-scalafix and/or Scala, or uninstalling sbt-scalafix plugin."
```

For the context, I find this confusing because
- As an sbt 1.x plugin, sbt-scalafix is only published with Scala 2.12.
- The constraint here is that for a given Scala 2.x, sbt-scalafix should use a scalafix version that uses a scalameta version for which the semanticdb compiler is published - this is very hard to check and almost equally hard to document.

Reply:
Ah, yes, you're right. It can be removed.